<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagalogDialog</class>
 <widget class="QDialog" name="TagalogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tagalog Dialog</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSelectFile">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>176</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File To Process</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFileName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>35</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>file name</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbTagalog">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tagalog</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbIlonggo">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>70</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilonggo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenAndProcess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>65</y>
     <width>266</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File And Process</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Verse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBookName">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>300</y>
     <width>366</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBookNumber">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblChapter">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVerse">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Verse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWebDB">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>105</y>
     <width>176</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select cWeb</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_cWEBDB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/drop-folder/Dropbox/NewBible/cWeb.bbl</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_TargetDB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/drop-folder/Dropbox/NewBible/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTargetDB">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>155</y>
     <width>176</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Target</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCmdFile">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>205</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Cmd File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_CmdFIle">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/drop-folder/Dropbox/NewBible/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cmds">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCreateChapterFiles">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>255</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Chapter Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReRun">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>205</y>
     <width>76</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>rerun</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>295</y>
     <width>156</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Bible FIle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStop">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>335</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Stop!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
